frontend/src/slices/auth/auth.slice.ts

```diff
@@ -48,13 +57,31 @@ const { reducer, actions, name } = createSlice({
     builder.addCase(getAuthenticatedUser.pending, (state) => {
       state.authenticatedUserDataStatus = DataStatus.PENDING;
     });
+
     builder.addCase(getAuthenticatedUser.fulfilled, (state, action) => {
       state.authenticatedUser = action.payload;
       state.authenticatedUserDataStatus = DataStatus.FULFILLED;
     });
+
     builder.addCase(getAuthenticatedUser.rejected, (state) => {
       state.authenticatedUserDataStatus = DataStatus.REJECTED;
     });
+
+    builder.addCase(createUserSurveyPreferences.pending, (state) => {
+      state.surveyPreferencesDataStatus = DataStatus.PENDING;
+    });
+
```

Comment:
remove spaces between blocks related to one action but with different states
```suggestion
```

Reply:
Done